atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -229,10 +249,70 @@ def replace_registry(self, image):
             return image
         return self._replace(image, registry=self.registry_replace[image.registry])
 
+    # TODO: rename, because it can also replace namespace
+    def replace_repo(self, image):
+        """
+        Replace image repo based on OSBS site/user configuration for image registry
+
+        :param image: ImageName
+        :return: ImageName
+        """
+        package_mapping = self._get_package_mapping(image.registry)
```

Comment:
We should avoid constructing this mapping in every call. This should be just one time operation.

Reply:
That should already be the case, `_get_package_mapping()` saves the constructed mapping in `self.package_mapping_by_registry` and takes it from there on subsequent calls